src/components/ReduxFetch.js

```diff
@@ -131,8 +133,40 @@ class ReduxFetch extends React.Component<Props, State> {
         }
       });
     }
+
+    if (onFulfilAll) {
+      const { allFetches: prevFetches } = this.createAllFetchesRepository(
+        prevProps.fetchData
+      );
+      const { allFetches } = this.createAllFetchesRepository(
+        this.props.fetchData
+      );
+      if (prevFetches.pending && allFetches.fulfilled) {
+        onFulfilAll();
+      }
+    }
   }
 
+  createAllFetchesRepository = (fetchData: Object) => {
+    const allPendingBools = map(
+      fetchData,
+      (repository: PromiseState<*>) => repository.pending
+    ).filter(
```

Comment:
I would skip the extra filtering, I guess it also works when doing something like:

```javascript
const aFetchIsPending = map(
      fetchData,
      (repository: PromiseState<*>) => repository.pending
    ).some(pendingBoolean => pendingBoolean === true);

return {
      allFetches: {
        fulfilled: !aFetchIsPending,
        pending: aFetchIsPending,
      },
    };
```